plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -883,6 +884,15 @@ public void setColumnComment(ConnectorSession session, JdbcTableHandle handle, J
         execute(session, sql);
     }
 
+    @Override
+    public String canonicalize(ConnectorSession session, String value, boolean delimited)
```

Comment:
I think this method should be implemented in base-jdbc module. It should behind feature toggle and if it is enabled it should disable (or not allow to set) `case-insensitive-name-matching`  and `case-insensitive-name-matching.config-file`.